corehq/apps/domain/templates/domain/partials/payment_methods.html

```diff
@@ -0,0 +1,149 @@
+              <br />
+              <span class="badge text-bg-primary">
+                {% trans "Autopay Card" %}
+              </span>
+            {% endif %}
+            {% if card.other_autopay_domains %}
+              <br />
+              <span class="badge text-bg-secondary">
+                {% trans "Autopay for Other Account" %}
+              </span>
+            {% endif %}
+          </td>
+          <td class="align-middle">************<strong>{{ card.last4 }}</strong></td>
+          <td class="align-middle">{{ card.exp_month }}/{{ card.exp_year }}</td>
+          <td class="align-middle">
+            {% if not card.is_autopay %}
+              <button
+                class="btn btn-outline-primary autopay-button"
+                type="button"
+                :disabled="isSubmitting"
+                @click="isSubmitting = true;"
+                hx-post="{{ request.path_info }}"
+                hq-hx-action="set_as_autopay"
+                hx-target="#manage-payment-methods-container"
+                hx-vals='{
+                  "token": "{{ card.token }}"
+                }'
+              >
+                {% trans "Set as autopay card" %}
+              </button>
+            {% endif %}
+          </td>
+          <td class="align-middle">
+            {% if not card.is_autopay and not card.other_autopay_domains %}
+              <button
+                class="btn btn-outline-danger"
+                type="button"
+                data-bs-toggle="modal"
+                data-bs-target="#remove-{{ card.token }}-modal"
+                :disabled="isSubmitting"
+                x-show="!isRemoving"
+              >
+                <i class="fa fa-remove"></i> {% trans "Remove Card" %}
+              </button>
+              <div
+                class="text-danger"
+                x-show="isRemoving"
+              >
+                <i class="fa fa-spin fa-spinner"></i> {% trans "Removing card..." %}
+              </div>
+            {% endif %}
+            <div
+              id="remove-{{ card.token }}-modal" class="modal fade"
+              tabindex="-1" aria-labelledby="remove-{{ card.token }}-modal-label" aria-hidden="true"
+            >
+              <div class="modal-dialog">
+                <div class="modal-content">
+                  <div class="modal-header">
+                    <h4 class="modal-title" id="remove-{{ card.token }}-modal-label">
+                      {% trans "Remove Card" %}
+                    </h4>
+                    <button
+                      class="btn-close" type="button" aria-label="Close"
+                      data-bs-dismiss="modal"
+                    ></button>
+                  </div>
+                  <div class="modal-body">
+                    {% blocktrans with brand=card.brand last4=card.last4 %}
+                      Actually remove <strong>{{ brand }}</strong>
+                      card ************<strong>{{ last4 }}</strong>?
```

Comment:
It seems this representation is used in at least 2 places now. Does it make sense to either expose this public representation directly on the card, and forego the selective bolding, or could we create a helper method for this?